includes/analytics/class-analytics.php

```diff
@@ -42,18 +44,25 @@ public function handle_event( string $name, array $context ) {
 			return;
 		}
 
-		wp_remote_post(
-			$this->event_url,
+		/**
+		 * Filters the URL used to send events to.
+		 *
+		 * @param string The URL to use when sending events.
+		 */
+		$url = apply_filters( 'stellarwp/analytics/event_url', self::DOMAIN . self::ENDPOINT );
+
+		$response = wp_remote_post(
```

Comment:
If we aren't returning the response here for verification of any kind (20x status code), we should make this non-blocking via `'blocking' => false` so it doesn't delay any execution of this being called.